Source/WebKit/webpushd/PushService.mm

```diff
@@ -55,32 +61,69 @@ static void updateTopicLists(PushServiceConnection& connection, PushDatabase& da
 
 #if HAVE(APPLE_PUSH_SERVICE_URL_TOKEN_SUPPORT)
 
+static void performAfterFirstUnlock(Function<void()>&& function)
+{
+#if PLATFORM(IOS) && USE(APPLE_INTERNAL_SDK)
+    if (MKBDeviceUnlockedSinceBoot()) {
+        function();
+        return;
+    }
+
+    RELEASE_LOG(Push, "Device is locked. Delaying init until it unlocks for the first time.");
+
+    static std::once_flag once;
+    static NeverDestroyed<Vector<Function<void()>>> functions;
+
+    std::call_once(once, [function = WTFMove(function)]() mutable {
```

Comment:
This does a WTFMove() of the function...

(See below)

Reply:
Good catch, I left this in from an earlier version of the patch and it's not necessary now (and will cause a crash).